src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3373,7 +3391,8 @@ private bool HasTopLevelNullabilityConversion(TypeSymbolWithAnnotations source,
                             checkConversion: false,
                             fromExplicitCast: false,
                             useLegacyWarnings,
-                            assignmentKind);
+                            assignmentKind,
+                            target);
```

Comment:
>target [](start = 28, length = 6)

It is hard to tell what scenario motivated this change. Do we have a unit-test that makes this change observable?

Reply:
Reverted.

---
In reply to: [241487312](https://github.com/dotnet/roslyn/pull/31703#discussion_r241487312) [](ancestors = 241487312)